test/index.test.js

```diff
@@ -74,25 +74,27 @@ test(file+" Force Fail in Email", function(t) {
   var person = {
     "name"     : "Bounce",
     "email"    : "invalid.email.address",
-    "subject":"Welcome to DWYL :)"
+    "subject"  : "Welcome to DWYL :)"
   };
   email('hello', person, function(err, data) {
     // console.log(' - - - - - - - - - - - ');
     // console.log(err, data);
     t.equal(err.statusCode, 400, "Invalid Mandrill Key");
-    t.end();
   })
+  t.end();
 });
 
-test(file+" send email (Success)", function(t) {
+test(file + " send email (Success)", function(t) {
   var person = {
-    name : "Success",
-    email: "success@simulator.amazonses.com",
-    "subject":"Welcome to DWYL :)"
+    name    : "Success",
+    email   : "success@simulator.amazonses.com",
+    subject : "Welcome to DWYL :)"
   }
+
   email('hello', person, function(err, data){
     // console.log(err, data);
+
     t.ok(data.MessageId.length > 0, 'Email Sent!');
-    t.end()
   })
+  t.end()
```

Comment:
You shouldn't put the `t.end` callback outside of the email function.
It will cause it to run immediately and not wait to test what is returned from the `email` function.